.github/workflows/tests.yml

```diff
@@ -4,7 +4,7 @@
 name: Tests
 
 on:
-  # pull_request:
+  pull_request:
```

Comment:
I've enabled the tests in this PR temporarily. Remember to revert this change before merging.
```suggestion
  # pull_request:
```

Reply:
Looks like those failures are gone.  The other failures are unrelated to this PR.